setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+packages = [
+    'robottelo',
+]
+
+requires = [
+    'ddt',
+    'fauxfactory',
+    'nailgun',
+    'paramiko',
+    'python-bugzilla',
+    'requests',
+    'selenium',
+]
+
+with open('README.rst', 'r') as f:
+    readme = f.read()
+
+setup(
+    name='robottelo',
+    version='0.1.0',
+    description='Robottelo is a test suite which exercises The Foreman.',
+    long_description=readme,
+    author=u'Satellite QE Team',
+    url='https://github.com/SatelliteQE/robottelo',
+    packages=packages,
+    package_data={'': ['LICENSE']},
+    package_dir={'robottelo': 'robottelo'},
+    include_package_data=True,
+    install_requires=requires,
```

Comment:
You can drop the list of packages from `requirements.txt` and have a single, canonical list of required packages. See how NailGun does it: https://github.com/SatelliteQE/nailgun/blob/master/requirements.txt.